src/components/proxy-middleware/rule_action_processor/processors/modify_response_processor.js

```diff
@@ -34,7 +34,10 @@ const process_modify_response_action = async (action, ctx) => {
 
       try {
         const parsedResponse = JSON.parse(finalBody)
-        if(action.responseType === GLOBAL_CONSTANTS.RESPONSE_BODY_TYPES.STATIC) {
+        if(
+          action.responseType === GLOBAL_CONSTANTS.RESPONSE_BODY_TYPES.STATIC || 
+          action.responseType === GLOBAL_CONSTANTS.RESPONSE_BODY_TYPES.LOCAL_FILE
+        ) {
```

Comment:
We can remove this check. Condition above this handle this aitomatically

Reply:
then this renders the PR as not required, so closing for now